network/wsNetwork_test.go

```diff
@@ -3757,3 +3759,106 @@ func TestWebsocketNetworkTelemetryTCP(t *testing.T) {
 	t.Log("closed detailsA", string(pcdA))
 	t.Log("closed detailsB", string(pcdB))
 }
+
+/*
+Exercises the updatePhonebookAddresses function, notably with different variations of valid relay and
+archival addresses.
+*/
+func TestUpdatePhonebookAddresses(t *testing.T) {
+	var netA *WebsocketNetwork
+
+	rapid.Check(t, func(t1 *rapid.T) {
```

Comment:
Is it correct to summarize the behavior here as: Check is called multiple times and the draw function is randomish, with some possible influence between runs due to the state in `t1`.

How can I tell how many times `Check` is called and what sort of influence the state has on the randomness?

Reply:
Also, it's not clear to me that there are actually duplicates being deduplicated. Should there be some assertions to make sure that the `len(primary) + len(secondary) < len(after)`